keyboards/frobiac/blackbowl/info.json

```diff
@@ -0,0 +1,84 @@
+{
+    "manufacturer": "frobiac",
+    "keyboard_name": "blackbowl",
+    "url": "https://www.github.com/frobiac/adnw",
+    "maintainer": "frobiac",
+    "bootloader": "halfkay",
+    "processor": "atmega32u4",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "dynamic_macro": true,
+        "extrakey": true,
+        "lto": true,
```

Comment:
this shold be under "build", IIRC

Reply:
On the branch I'm on in all `info.json`s:
* lto is under build, correct
* dynamic_macro isn't used by anyone
* console and extrakey seem to be below "features" everywhere.
I will compare to recent changes and docs after reviewing the remainder of your suggestions.

Reply:
Current master branch has the same setup, and all but "lto" cause compile errors when put under "build" instead of "features":

```
QMK Firmware 0.19.12
☒ Not including data from file: keyboards/frobiac/blackbowl/info.json
☒ 	build: Additional properties are not allowed ('console', 'dynamic_macro', 'extrakey' were unexpected)
```